ods_ci/libs/DataSciencePipelinesAPI.py

```diff
@@ -300,8 +299,9 @@ def get_default_storage(self):
         result, _ = self.run_oc("oc get storageclass -A -o json")
         result = json.loads(result)
         for storage_class in result['items']:
-            if storage_class['metadata']['annotations']['storageclass.kubernetes.io/is-default-class'] == 'true':
-                break
+            if 'annotations' in storage_class['metadata']:
```

Comment:
Shouldn't you also check if  storage_class is null/empty?

Reply:
so far, no.